Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -156,6 +156,18 @@ def GetDscLibrarys (self):
         ]
         return dsc_libs
 
+    def GetKeyHashList (self):
+        # Define a set of new key used for different purposes
+        # The key is either public key PEM format or private key PEM format
+        pub_key_list = [
+          (
+            # Use a single test key
+            HASH_USAGE['PUBKEY_CFG_DATA'] | HASH_USAGE['PUBKEY_FWU'] | HASH_USAGE['PUBKEY_OS'] | HASH_USAGE['PUBKEY_CONT_DEF'],
+            'TestSigningPrivateKey.pem'

```

Comment:
Question: Does GetKeyHashList override keys defined in BuildLoader.py?

Reply:
Not override.  But append.